airflow/ui/src/pages/Dashboard/Stats/DAGImportErrorsModal.tsx

```diff
@@ -0,0 +1,125 @@
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Heading, Text, HStack, Input } from "@chakra-ui/react";
+import { useEffect, useState } from "react";
+import { PiFilePy } from "react-icons/pi";
+
+import type { ImportErrorResponse } from "openapi/requests/types.gen";
+import Time from "src/components/Time";
+import { Accordion, Dialog } from "src/components/ui";
+import { Pagination } from "src/components/ui/Pagination";
+
+type ImportDAGErrorModalProps = {
+  importErrors: Array<ImportErrorResponse>;
+  onClose: () => void;
+  open: boolean;
+};
+
+const PAGE_LIMIT = 5;
+
+export const DAGImportErrorsModal: React.FC<ImportDAGErrorModalProps> = ({
+  importErrors,
+  onClose,
+  open,
+}) => {
+  const [page, setPage] = useState(1);
+  const [searchQuery, setSearchQuery] = useState("");
+  const [filteredErrors, setFilteredErrors] = useState(importErrors);
+
+  const startRange = (page - 1) * PAGE_LIMIT;
+  const endRange = startRange + PAGE_LIMIT;
+  const visibleItems = filteredErrors.slice(startRange, endRange);
+
+  const onOpenChange = () => {
+    setSearchQuery("");
+    setPage(1);
+    onClose();
+  };
+
+  useEffect(() => {
+    const query = searchQuery.toLowerCase();
+    const filtered = importErrors.filter((error) =>
+      error.filename.toLowerCase().includes(query),
+    );
+
+    setFilteredErrors(filtered);
+    setPage(1);
+  }, [searchQuery, importErrors]);
+
+  return (
+    <Dialog.Root
+      onOpenChange={onOpenChange}
+      open={open}
+      scrollBehavior="inside"
+      size="xl"
+    >
+      <Dialog.Content backdrop>
+        <Dialog.Header>
+          <Heading size="xl">Dag Import Errors</Heading>
+          <Input
+            mt={4}
+            onChange={(letters) => setSearchQuery(letters.target.value)}
+            placeholder="Search by file"
+            value={searchQuery}
+          />
+        </Dialog.Header>
+
+        <Dialog.CloseTrigger />
+
+        <Dialog.Body>
+          <Accordion.Root collapsible multiple size="md" variant="enclosed">
+            {visibleItems.map((importError) => (
+              <Accordion.Item
+                key={importError.import_error_id}
+                value={importError.filename}
+              >
+                <Accordion.ItemTrigger cursor="pointer">
+                  <PiFilePy />
+                  {importError.filename}
+                </Accordion.ItemTrigger>
+                <Accordion.ItemContent>
+                  <Text color="fg.muted" fontSize="sm" mb={1}>
+                    Timestamp: <Time datetime={importError.timestamp} />
+                  </Text>
+                  <Text color="fg.error" fontSize="sm" whiteSpace="pre-wrap">
+                    <code>{importError.stack_trace}</code>
+                  </Text>
```

Comment:
```suggestion
                  <Code color="fg.error" fontSize="sm" whiteSpace="pre-wrap">
                    {importError.stack_trace}
                  </Code>
```

We can just use chakra's `Code` component

Reply:
<img width="996" alt="image" src="https://github.com/user-attachments/assets/0064184d-54bb-4eb8-9ac0-149b0a02cab3">
In that case, we need to handle the bg color of code block or change variant of Accordion.root to be subtle, which removes the border though.